### PR TITLE
TK274 -

### PR DESCRIPTION
**Pull request by karuna835**
- Reinstate option for Alumni status.

### CODE REVIEW

File: app/apis/btcapi/subs/edit.php [hunk 1]
```diff
@@ -22,6 +22,15 @@
...
 	$subs = $_POST["subs"];
+	//TK274- Checked end_date set or not.
+	if(isset($_POST['end_date']))
+	{
+		$end_date = $_POST['end_date'];
+	}
+	else
+	{
+		$end_date = "";
+	}
```
Comment (kailasb) [inline app/apis/btcapi/subs/edit.php:33]: user ternary operator

File: b/js/app.js [hunk 2]
```diff
@@ -1140,7 +1146,15 @@ angular.module("batcave", ["ngRoute","angularjs-dropdown-multiselect"])
 			$scope.assignments = $scope.user.assignments[courseCode];
 			angular.element("#assignments").modal("show");
 		}
+		if($scope.user.from_date && $scope.user.from_date.length > 0)
+						{
+							$scope.isDisabled = true;
+						}
+						else{
+							$scope.isDisabled = false;
+						}
```
Comment (kailasb) [inline b/js/app.js:1155]: use ternary operator

File: b/js/app.js [hunk 3]
```diff
@@ -1353,14 +1367,26 @@ angular.module("batcave", ["ngRoute","angularjs-dropdown-multiselect"])
...
+		$scope.clickDate = function() {
+			$scope.user.end_date = new Date();
+			if(angular.isDate($scope.user.end_date) == true)
+			{
+				$scope.isValue = true;
+			}
+			else
+			{
+				$scope.isValue = false;
+			}
```
Comment (kailasb) [inline b/js/app.js:1388]: use ternary operator

**Review by kailasb — CHANGES_REQUESTED** [review]
check the feedback

File: app/apis/btcapi/subs/edit.php (hunk 1 above)
Comment (karuna835) [reply app/apis/btcapi/subs/edit.php]: I have used ternary operator

File: b/js/app.js (hunk 2 above)
Comment (karuna835) [reply b/js/app.js]: I have used ternary operator

File: b/js/app.js (hunk 3 above)
Comment (karuna835) [reply b/js/app.js]: I have used ternary operator

**Review by kailasb — APPROVED** [review]
changes are proper

**Pull request merged**